uninstall-hook.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# uninstall-hook.sh
+# Used to reverse the modifications made for the pre-commit hook by setup-repo.sh
```

Comment:
"Used to remove the git precommit hook for this repository (created by setup-repo.sh)."